src/RadixDlt.NetworkGateway.PostgresIntegration/Metrics/SqlQueryMetricsHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+
+namespace RadixDlt.NetworkGateway.PostgresIntegration.Metrics;
+
+public static class SqlQueryMetricsHelper
+{
+    public const string QueryNameTag = "QueryName";
+
+    public static string GetQueryNameValue(
```

Comment:
1. Why are those arguments optional and default to empty string? Suspicious.
2. Do we really need filename? It's a heavy operation, it's not cached, it seems bit pointless after all. Method name or even type name + method name should be sufficient, don't you think?